phoebe/frontend/default_bundles/default_star.bundle

```diff
@@ -863,12 +863,11 @@ null
 "context": "compute",
 "description": "Atmosphere table",
 "choices": [
-"phoenix",
 "extern_atmx",
-"extern_planckint",
-"blackbody",
+"ck2004",
```

Comment:
all the changes to this file are inconsequential; I'd leave the original version.